source/NVDAObjects/window/winword.py

```diff
@@ -558,6 +558,31 @@ def collectionFromRange(self,rangeObj):
 	def filter(self,item):
 		return item.type==wdInlineShapeChart
 
+
+class LazyControlField(textInfos.ControlField):

```

Comment:
Given this is just for row and column header text values, consider renaming it to make it clear eg `LazyControlField_RowAndColumnHeaderText` verbose, but captures the specific nature of what this does. At the moment this class sounds more generic than it is.


`Lazy fetching of row and column header text values`